<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateManualEulerRingL1Dialog</class>
 <widget class="QDialog" name="CreateManualEulerRingL1Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Manual EulerRingL1</string>
  </property>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyEulerRingL1.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditItemName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ButtonOK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyEulerRingL1.qrc">
     <normaloff>:/Qtres/OK.png</normaloff>:/Qtres/OK.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>20</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>検
査</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>1</number>
   </property>
  </widget>
  <widget class="QFrame" name="frameLibFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Item name</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidgetInspect">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Ring</string>
    </attribute>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>暗側</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>OK輝度範囲</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>明側</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditDarkWidth">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditLightWidth">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>接続距離</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditConnectLen">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnitName_8">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxConnectLenMM">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnitName_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkModeEnableOpenRingCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>98</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>座切れ検査を実行</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Hole</string>
    </attribute>
    <widget class="QSpinBox" name="EditMinHoleDiameter">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>最小穴径</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditMaxShiftHole">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>穴移動範囲</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMinHoleDiameterMM">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxShiftHoleMM">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnitName_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxHoleDiameterMM">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditMaxHoleDiameter">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>最大穴径</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>参考輝度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditHoleBrightnessAsReference">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>属性</string>
    </attribute>
    <widget class="QCheckBox" name="checkModeEnabled">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>検査有効</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkModeAbsoluteBright">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>絶対輝度</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkModeCenterBrightFromParts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>211</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>部品から輝度を抽出する</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkModeDynamicMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>実行マスク生成</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkModeEnableHoleCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>穴判定を行う</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>補正</string>
    </attribute>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>明側　最大補正輝度量</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>暗側　最大補正輝度量</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditAdjustWhite">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="EditAdjustBlack">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>マスク生成</string>
    </attribute>
    <widget class="QSpinBox" name="EditExpandForDynamicMask">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>マスク膨張ドット</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="PropertyEulerRingL1.qrc"/>
 </resources>
 <connections/>
</ui>
